libraries/util/profile/myBeans.py

```diff
@@ -0,0 +1,49 @@
+import libraries.page.profile.myBeans as pMyBeans
+import libraries.page.common.adminKpc as pAdminKpc
+
+import libraries.util.common as uCommon
+import libraries.util.appCommon.appComm as uAppComm
+import libraries.util.appCommon.adminKpc as uAdminKpc
+
+class com:
+    """COMMON"""
+    @uCommon.ufuncLog   
+    def verifyBeansRewardFromAddingChild(page):
+        """ 
+        Objective: To check if On Add Child was rewarded.
+        param: None
+        returns: None
+        Author: abernal_20240229
+        """  
+        uCommon.waitElemToBeVisible(page, pMyBeans.com.onAddChildLbl)
+        
+    @uCommon.ufuncLog   
+    def verifyBeansRewardFromAddingAttribute(page):
+        """ 
+        Objective: To check if On Add Attribute was rewarded.
+        param: None
+        returns: None
+        Author: abernal_20240229
+        """  
+        uCommon.waitElemToBeVisible(page, pMyBeans.com.onAddAttributeLbl)
```

Comment:
Same comment as above. 
Would it make sense if either expectElemToBeVisible or verifyVisible is used instead of waitElemToBeVisible?
verifyVisible - checking if visible or not returns True or False. From this results you can add steps what the system will prompt based on the result.

expectElemToBeVisible - with appropriate log msg when elem is visible.